### DISCUSSION TITLE
Rosetta Code examples

### DISCUSSION THREAD

**Discussion by uvtc** [janet-lang/janet#948] [General]
Just posting to say that when anyone adds a Janet example to [Rosetta Code](http://rosettacode.org/), it would nice to hear about it here in the forum.

**Comment by sogaiu** [comment]
For historical reference, there is a record of some activity along tthese lines here: https://github.com/janet-lang/janet/issues/227

**Comment by Andriamanitra** [comment]
I just added examples for [Count in octal](https://rosettacode.org/wiki/Count_in_octal#Janet) and [Dot product](https://rosettacode.org/wiki/Dot_product#Janet).

**Comment by eimrine** [comment]
Any list of examples with not just Janet code but with Janet C modules?

**Reply by sogaiu** [reply]
I don't know of a list that's filtered for Janet C modules.

There is however, [this list of repositories](https://github.com/janet-lang/janet/files/11039720/repositories.txt).

**Reply by sogaiu** [reply]
@eimrine I looked for `declare-native` across those repositories and the result was the following.
<details>
<pre>
https://github.com/ahungry/ahungry-janet/
https://github.com/ahungry/janet-code/
https://github.com/ahungry/janet-p99-map/
https://github.com/ahungry/janet-pobox/
https://github.com/ahungry/janet-sig-handlers/
https://github.com/ahungry/janet-xbuild/
https://github.com/ahungry/puny-gui/
https://github.com/ahungry/puny-server/
https://github.com/AlecTroemel/janet-chipmunk/
https://github.com/AlecTroemel/janet-physac/
https://github.com/Alligator/termbox-janet/
https://github.com/andrewchambers/hermes/
https://github.com/andrewchambers/hpkg/
https://github.com/andrewchambers/janet-bare1/
https://github.com/andrewchambers/janet-base16/
https://github.com/andrewchambers/janet-big/
https://github.com/andrewchambers/janet-bytefmt/
https://github.com/andrewchambers/janet-ctrl-c/
https://github.com/andrewchambers/janet-flock/
https://github.com/andrewchambers/janet-jcjit/
https://github.com/andrewchambers/janet-logfmt/
https://github.com/andrewchambers/janet-pico-http-parser/
https://github.com/andrewchambers/janet-posix-spawn/
https://github.com/andrewchambers/janet-pq/
https://github.com/andrewchambers/janet-process/
https://github.com/andrewchambers/janet-redis/
https://github.com/andrewchambers/janet-rlrepl/
https://github.com/andrewchambers/janet-sh/
https://github.com/andrewchambers/janet-uri/
https://github.com/andrewchambers/janet-utf8/
https://github.com/andrewchambers/janet-where-defined/
https://github.com/andrewchambers/janet-yacc/
https://github.com/andrewchambers/poolparty/
https://github.com/bakpakin/janet-miniz/
https://github.com/bunder/janet-primesieve/
https://github.com/cellularmitosis/janet-lzo/
https://github.com/CFiggers/janet-termios/
https://github.com/corasaurus-hex/isatty/
https://github.com/dalygbarron/pingo/
https://github.com/dracconi/yanka/
https://github.com/good-place/tahani/
https://github.com/GrayJack/janet-str-ext/
https://github.com/GrayJack/rust-janet-module-template/
https://github.com/ianthehenry/janet-clipper/
https://github.com/ianthehenry/jimmy/
https://github.com/ishehadeh/janet-format/
https://github.com/janet-lang/circlet/
https://github.com/janet-lang/jaylib/
https://github.com/janet-lang/jhydro/
https://github.com/janet-lang/json/
https://github.com/janet-lang/juv/
https://github.com/janet-lang/spork/
https://github.com/janet-lang/sqlite3/
https://github.com/janet-lang/tarray/
https://github.com/janet-lang/webview/
https://github.com/joy-framework/codec/
https://github.com/joy-framework/halo/
https://github.com/joy-framework/http/
https://github.com/joy-framework/moondown/
https://github.com/joy-framework/uuid/
https://github.com/kixus/mycom-mygreetlib/
https://github.com/LeviSchuck/janetls/
https://github.com/llmII/jsys/
https://github.com/MikeBeller/janet-abstract/
https://github.com/MikeBeller/janet-benchmarksgame/
https://github.com/MorganPeterson/jermbox/
https://github.com/MorganPeterson/jlzf/
https://github.com/nate/isatty/
https://github.com/newhook/janet-mysql/
https://github.com/pauldub/janet-http-client/
https://github.com/pingiun/jequests/
https://github.com/pyrmont/claret/
https://github.com/pyrmont/ecstatic/
https://github.com/pyrmont/markable/
https://github.com/pyrmont/outfool/
https://github.com/pyrmont/persimmon/
https://github.com/pyrmont/settee/
https://github.com/pyrmont/watchful/
https://github.com/rushsteve1/spinnerette/
https://github.com/saikyun/freja-jaylib/
https://github.com/saikyun/janet-binary-search-comparison/
https://github.com/saikyun/janet-whereami/
https://github.com/saikyun/text-experiment/
https://github.com/scotthaleen/jhttp/
https://github.com/sepisoad/jtbox/
https://github.com/sepisoad/jurl/
https://github.com/sevanteri/janet-secret/
https://github.com/Techcable/janet-msgpack/
https://github.com/thegeez/janet-postings/
https://github.com/tionis/dice.janet/
https://github.com/yumaikas/anno/
https://github.com/yumaikas/chrono/
https://git.sr.ht/~bakpakin/jvk/
https://git.sr.ht/~bakpakin/moogtool/
https://git.sr.ht/~pepe/best-self/
https://git.sr.ht/~pepe/gp/
https://git.sr.ht/~pepe/jfzy/
https://git.sr.ht/~pepe/jpid/
https://git.sr.ht/~pepe/shriek/
https://git.sr.ht/~subsetpark/janet-rl/
https://git.sr.ht/~subsetpark/jnj/
https://tasadar.net/tionis/glyph.git
</pre>
</details>

May be that's a little less of a haystack...
